### PR TITLE
Introduction to testbeam infrastructure

### PR DESCRIPTION
**Pull request by cbespin**
This PR adds an introduction to possible testbeam topics (tbd at a later point/new PR)

### CODE REVIEW

File: intros/testbeam.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Beam Test Infrastructure
+
+***
+
+![Beam telescope](/imgs/beam_telescope.jpg)
+Dedicated beam test campaigns are regularily conducted by our group to measure the hit detection efficiency of pixel detectors.
```
Comment (leloup314) [inline intros/testbeam.md:7]: newline pls

File: intros/testbeam.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+This measurements needs a high-energy particle beam provided by accelerators and an array of detector planes to identify unique particle tracks.
+With that information one can confirm if the device under test (a so called DUT) also registered a hit at that time and at the predicted location.
+
+The required setups consist of many devices and components of which a lot are developed and built by our group.
```
Comment (leloup314) [inline intros/testbeam.md]: newline + *** pls

**Review by leloup314 — CHANGES_REQUESTED** [review]

File: intros/testbeam.md (hunk 1 above)
Comment (cbespin) [reply intros/testbeam.md:7]: Missed that, sorry

File: intros/testbeam.md (hunk 2 above)
Comment (cbespin) [reply intros/testbeam.md]: Also done now

**Comment by cbespin** [comment]
Formatting should be consistent now

**Comment by leloup314** [comment]
Beautiful

**Pull request merged**